app/src/test/java/com/locoquest/app/BenchmarkUnitTests.kt

```diff
@@ -61,8 +61,9 @@ class BenchmarkServiceTest {
                 elevation = 8848.86,
                 description = 0.0,
                 d = 0.0,
-                d1 = 0.0))
-            return benchmark
+                d1 = 0.0
+            )
+
         }
     }
 }
```

Comment:
Changed Mockbenchmark service back to the original after testing code. Disregard as well